indra/llui/lltexteditor.cpp

```diff
@@ -1136,6 +1140,8 @@ S32 LLTextEditor::addChar(S32 pos, llwchar wc)
 		S32 delta = 0;
 		if (mPrevalidateFunc)
 		{
+			LLUI::getInstance()->reportBadKeystroke();
```

Comment:
This looks like it would pop up on every validating case.
I think such message would be usefull when trying to input utf-8 into ansi-only string, but there is no point showing it for something like validateNonNegativeS32 which obviously only accepts nubers.